frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import {
+  StateCounts,
+  Severity,
+  AlertState,
+  alertsRowFilters,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import { alertDescription, alertState, alertSeverity } from '@console/internal/reducers/monitoring';
+import { Alert, Rule } from '@console/internal/components/monitoring/types';
+import { YellowExclamationTriangleIcon } from '@console/shared';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+export const monitoringAlertColumn = [
+  {
+    title: 'Name',
+    cellFormatters: [expandable],
+    transforms: [sortable],
+  },
+  { title: 'Severity', transforms: [sortable, cellWidth(20)] },
+  { title: 'Alert State', transforms: [sortable, cellWidth(20)] },
+  '',
+];
+
+export const monitoringAlertRows = (alertrules: Rule[]) => {
+  const rows = [];
+  _.map(alertrules, (rls) => {
```

Comment:
Nit: Since we're not using the return value, probably better to use `_.each` than `_.map`. Alternatively, it could be changed so that `monitoringAlertRows` returns the value from `_.map`.